tamr_unify_client/client.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         port: Optional[int] = 9100,
         base_path: str = "/api/versioned/v1/",
         session: Optional[requests.Session] = None,
+        store_auth_cookie: bool = False,  # TODO: Remove this - stopgap to pass existing tests
```

Comment:
@pcattori What might be a better solution here?

Reply:
This sort of duct-tape solution is fine for TUC since its underlying implementation makes this sort of thing hard anyway.